README.md

```diff
@@ -13,14 +13,12 @@ The `safe-config-service` is a service that provides configuration information i
 
 ### 1. Configuration
 
-The environment variables are set via the `.env` file. The configuration in `.env.example` is meant to be production ready. You can copy it and adjust it to your development needs (refer to the file for the explanation about each environment variable).
-
+The environment variables are set via the `.env` file. The configuration in `.env.example` is meant to be development ready. You can copy it and adjust it to your development needs (refer to the file for the explanation about each environment variable).
 ```shell
 cp .env.example .env
 ```
 
-**Important:** Some variables are required to be set before running the application: `SECRET_KEY`, `POSTGRES_USER`, `POSTGRES_PASSWORD`.
-They can be set either locally on your environment or (as a provided example) by uncommenting these variables from the `.env` file.
+**Important:** It's recommended change properly passwords and another default parameters for production environment.
```

Comment:
nitpick: some types but also generalising to any field.

```suggestion
**Important:** It is recommended to set sensitive fields (like passwords) and adjust other parameters for your production environment needs. The `.env.sample` configuration shouldn't be used for any production deployment.
```

Reply:
Yes It is more clear. :)